client/boot/index.js

```diff
@@ -52,7 +52,6 @@ var config = require( 'config' ),
 	supportUser = require( 'lib/user/support-user-interop' ),
 	isSectionIsomorphic = require( 'state/ui/selectors' ).isSectionIsomorphic,
```

Comment:
You could move this selector import to the new  `import { ... } from 'state/ui/selectors'` statement.